database/migrations/0060-update_communities_table.ts

```diff
@@ -0,0 +1,32 @@
+/* MIGRATION TO CREATE THE FEDERATION COMMUNITY TABLES
+ *
+ * This migration creates the `community` and 'communityfederation' tables in the `apollo` database (`gradido_community`).
+ */
+
+/* eslint-disable @typescript-eslint/explicit-module-boundary-types */
+/* eslint-disable @typescript-eslint/no-explicit-any */
+
+export async function upgrade(queryFn: (query: string, values?: any[]) => Promise<Array<any>>) {
+  await queryFn(
+    'ALTER TABLE `communities` MODIFY COLUMN `last_announced_at` datetime(3) AFTER `end_point`;',
```

Comment:
In your other PR you have the same migration but without `last_announced_at` - something seems fishy here.

Reply:
its not fishy than its more tricky to keep several PR in sync without knowing which one will reach an approve to be earlier than the other...
the first merged PR wins and the follower PRs have to be adapted on the masters state...